internal/controllers/analyzer/analyzer.go

```diff
@@ -172,20 +172,33 @@ func (a *Analyzer) runAnalysis() (totalVulns int, err error) {
 		return 0, err
 	}
 	a.startDetectVulnerabilities(langs)
-	return a.sendAnalysisAndStartPrintResults()
+	if err = a.sendAnalysis(); err != nil {
+		//TODO: just log the error?
+		logger.LogError("error sending analysis ", err)
```

Comment:
The `HorusecService` already log the error, I think that we can move these logs from service to here.